src/main/java/bio/terra/stairway/FlightDao.java

```diff
@@ -841,6 +891,30 @@ private void storeInputParameters(
     }
   }
 
+  private void storeWorkingParameters(
+      Connection connection, String flightId, Instant logTime, FlightMap workingParameters)
+      throws SQLException {
+    List<FlightInput> inputList = workingParameters.makeFlightInputList();
+
+    final String sqlInsertInput =
+        "INSERT INTO "
+            + FLIGHT_WORKING_TABLE
+            + " (flightId, log_time, key, value) VALUES (:flightId, :logTime, :key, :value)";
```

Comment:
Is the log_time in FLIGHT_WORKING_TABLE acting as a foreign key for the FLIGHT_LOG_TABLE? This feels sneaky and surprising. Should we just use a unique FLIGHT_LOG_TABLE key as a foreign key instead?

Reply:
I like that idea.

Reply:
Sure, this raises a question with using this table for output parameters though. If we clean up the logs we will lose the last set of working parameters which we now look at for output parameters.  Is that acceptable?

Reply:
We discussed just using flightworking for output parameters and removing the code that deletes FlightLog entries, so the last flightworking will always be available.